telco-hub/configuration/reference-crs/required/gitops/ex-ztp-repo-ssh.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ztp-repo
+  namespace: openshift-gitops
+  labels:
+    argocd.argoproj.io/secret-type: repository
+stringData:
+  url: git@gitlab.example.com:namespace/repo.git
+  insecure: "false"
+  name: ztp
+  project: default
```

Comment:
Should this be ztp-app-project to match the AppProject below?

Reply:
I am trying to find out why the key Project here. Neither on the doc:
https://argo-cd.readthedocs.io/en/stable/user-guide/private-repositories/
nor examples:
https://argo-cd.readthedocs.io/en/stable/operator-manual/argocd-repositories-yaml/
is been used. 
I think never set it, and worked oka. So, I suggest to get rid of it. At least, until we know it is needed for any reason.